InformationScripting/src/graph/Property.h

```diff
@@ -41,11 +43,16 @@ class INFORMATIONSCRIPTING_API Property {
 		friend boost::python::object pythonObject(const Property& p);
 
 		template <class ConvertTo> inline operator ConvertTo() const;
+		inline operator Model::Node*() const;
+
+		bool operator==(const Property& other) const;
 
 	private:
 		struct PropertyDataConcept {
 				virtual ~PropertyDataConcept() = default;
 				virtual boost::python::object pythonObject() const = 0;
+				virtual Model::Node* node() const { return nullptr; }
+				virtual bool equals(std::shared_ptr<PropertyDataConcept> other) const = 0;
```

Comment:
why not take a `&` to `other`. This will make the method faster and we don't really need to take ownership for it.